docs/develop/sdk/1-getting-started-guide/00-getting-started.md

```diff
@@ -0,0 +1,237 @@
+
+Connecting and disconnecting to the KILT is an important step requiring a setup of the configurations.
+The connection allows to **query**, **verify** and **interact** with the chain.
+
+**Lets get connected.**
+
+Inside the `getting-started.js` inside the `main` function, you will need to begin by connecting to a **KILT node**.
+Using the imported SDK, it exposes **`Kilt.init()`** to initalise the connection to the KILT blockchain via an address.
+
+We will initalise the **KILT blockchain** named the **Spiritnet**.  
+
+``` js
+    ...
+async function main() {
+    await Kilt.init({address: 'wss://spiritnet.kilt.io/'})
+...
+```
+
+Now you have initalised the connection, lets connect to the chain.
+Inside the `main` function, lets get the conncetion using an asynchronous call that creates a connection, or checks if an existing connection is available and connects directly.
+
+``` js
+...
+    await Kilt.init({address: 'wss://spiritnet.kilt.io/'})
+    
+    await Kilt.ChainHelpers.BlockchainApiConnection.getConnectionOrConnect()
+...
+```
+
+Now you have connected you have access to the chain, but lets not forget to **close** any connections.
+Its best practice not to leave an connections open, add `Kilt.disconnect()` at the bottom of `main` function.
+
+``` js
+...
+    await Kilt.disconnect()
+    return
+...
+```
+
+The `Kilt.disconncet()` will close any established connections.
+
+**Congratulations!
+You have done connected, lets start querying and fetching data from the chain!**
+
+## Lets query the chain
+
+Now you have a connection to the chain to query a variety of information using the exposed KILT SDK functionality.
+We will be looking at the **web3names** (`john_doe`) and using them to fetch the corresponding **DID identifier**.
+Underneath the blockchain connection, add the following lines.
+
+``` js
+...
+    await Kilt.ChainHelpers.BlockchainApiConnection.getConnectionOrConnect()
+
+    const johnDoeDidId = await Kilt.Did.Web3Names.queryDidForWeb3Name('john_doe')
+    
+    console.log(`Hello world, my name is john_doe and this is my DID ${johnDoeDidId}`)
+...
+```
+
+Try excuting it and see what comes out.
+
+Did you get the DID? Nice work! You have `john_doe`'s DID.
+Now let's see if they have any credentials availale to verify!
+
+## Verifying a credential
+
+**DIDs** are great tools for identification.
+A DID can have service endpoints on the DID linking to public data.
+Therefore, you can take your credentials and display them for others to see!
+Lets see how we can check a **DIDs endpoints** and see if `john_doe` has any public credentials for us to **query** and **verify**.
+
+Lets take the DID that was fetch and see if we can retreives the contents.
+We will add a new line under the `console.log` and lets resolve the DID with the **DID identifier** fetched with the **web3name**.
+
+``` js
+...
+    console.log(`Hello world, my name is john_doe and this is my DID ${johnDoeDidId}`)
+    
+    if (!johnDoeDidId) return 
```

Comment:
maybe we also should have a message in the error case? And the `hello world thing should be printed after we checked that he exists?

```suggestion
    if (!johnDoeDidId) return console.log('john_doe doesn't exist')
    
    console.log(`Hello world, my name is john_doe and this is my DID ${johnDoeDidId}`)
```

Reply:
Updated